persist-admin-notices-dismissal.php

```diff
@@ -100,9 +100,13 @@ public static function dismiss_admin_notice() {
 		 * @return bool
 		 */
 		public static function is_admin_notice_active( $arg ) {
-			$array       = explode( '-', $arg );
-			$length      = array_pop( $array );
-			$option_name = implode( '-', $array );
+			if (strpos($arg,'-')) {
```

Comment:
Well, I missed that requirement :-) Because the javascript works just fine with any value (like I'm using it for ex. eme_hello_notice-forever), and in php array_pop pops from the end of an array, so there the "data-" is not really required either (it would work for "eme_hello_notice-forever" too IMHO)